app/assets/javascripts/modules/userChoiceCtrl.js

```diff
@@ -0,0 +1,43 @@
+angular.module('probedock.userChoiceCtrl', [ 'probedock.api' ])
+  .directive('userChoiceCtrl', function() {
+    return {
+      restrict: 'E',
+      controller: 'UserChoiceCtrl',
+      templateUrl: '/templates/user-choice-ctrl.html',
+      scope: {
+        organization: '=',
+        update: '='
```

Comment:
There's a dedicated way of passing functions to directives which reduces the coupling with the parent scope. Use `&` instead of `=` and call it `onUpdate` (convention).